src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -12,7 +12,6 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Configuration" />
```

Comment:
@SimonCropp This reference cannot be removed. The licensing code also requires this.